src/Tools/RepoUtil/NuGetFeed.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace RepoUtil
+{
+    internal struct NuGetFeed : IEquatable<NuGetFeed>
+    {
+        internal string Name { get; }
+        internal Uri Url { get; }
+
+        internal NuGetFeed(string name, Uri url)
+        {
+            Name = name;
+            Url = url;
+        }
+
+        public static bool operator ==(NuGetFeed left, NuGetFeed right) =>
+            Constants.NugetPackageNameComparer.Equals(left.Name, right.Name) &&
+            Constants.NugetPackageVersionComparer.Equals(left.Url, right.Url);
```

Comment:
These comparers are specific to NuGet packages.  For these entries I think we need to use different comparers:

1. Name: `StringComparer.OrdinalIgnoreCase`.  Don't have a strong opinion here but doesn't seem like name casing should matter. 
2. I'd just compare the URI directly as they support equality (uri equality is compilcated).